contracts/icp/proxy-contract/src/lib.rs

```diff
@@ -14,7 +14,7 @@ pub mod mutate;
 pub mod query;
 pub mod types;
 
-#[derive(CandidType, Serialize, Deserialize, Default)]
+#[derive(Serialize, Deserialize, Default)]
```

Comment:
everything but `Default` can be removed, yeah?

```suggestion
#[derive(Default)]
```